### DISCUSSION TITLE
Bora conversar sobre arquitetura frontend na Octogatos Conf 2022!

### DISCUSSION THREAD

**Discussion by woliveiras** [gittogethers/octogatosconf22#86] [Where in the world?]
🇧🇷 Versão em Português

Olá, pessoas!

Eu vou fazer uma palestra sobre **Arquitetura frontend para aplicações de alta escalabilidade** na nossa Octogatos Conf 2022. 

Quais suas dúvidas sobre arquitetura frontend?

Também me siga no [Twitter](https://twitter.com/1ilhas) para conversarmos mais sobre esse tema. 

--- 

🇬🇧 English Version

Hello people!

I'm going to talk about **Frontend architecture for highly scalable applications** at our Octogatos Conf 2022.

What are your questions about frontend architecture?

Also, follow me on [Twitter](https://twitter.com/1ilhas) to talk more about this topic.

---

![Oliveira_-_Arquitetura_frontend_para_aplicacoes_de_alta_escalabilidade_335227](https://user-images.githubusercontent.com/4243601/188149075-10c82020-3818-461e-b027-b52625a42a11.jpeg)

**Comment by AndreaGriffiths11** [comment]
Welcome William! I look forward to your talk!
